net-analyzer/netdata/netdata-1.6.0.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit autotools fcaps linux-info python-r1 systemd user
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="git://github.com/firehol/${PN}.git"
+	inherit git-r3
+	SRC_URI=""
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/firehol/netdata/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Linux real time system monitoring, done right!"
+HOMEPAGE="https://github.com/firehol/netdata https://my-netdata.io/"
+
+LICENSE="GPL-3+ MIT BSD"
+SLOT="0"
+IUSE="caps +compression cpu_flags_x86_sse2 ipmi mysql nfacct nodejs postgres +python"
+REQUIRED_USE="
+	mysql? ( python )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+# most unconditional dependencies are for plugins.d/charts.d.plugin:
+RDEPEND="
+	>=app-shells/bash-4:0
+	net-misc/curl
+	net-misc/wget
+	virtual/awk
+	|| ( net-analyzer/netcat6 net-analyzer/netcat )
+	net-analyzer/tcpdump
+	net-analyzer/traceroute
+	caps? ( sys-libs/libcap )
+	compression? ( sys-libs/zlib )
+	ipmi? ( sys-libs/freeipmi )
+	sys-apps/util-linux
+	python? (
+		${PYTHON_DEPS}
+		dev-python/pyyaml[${PYTHON_USEDEP}]
+		mysql? (
+			|| ( dev-python/mysqlclient[${PYTHON_USEDEP}] dev-python/mysql-python[${PYTHON_USEDEP}] )
```

Comment:
have you checked which one is the better one? I remember that one of them is quasi-defunct, but dont recall which